itests/org.jboss.tools.cdk.ui.bot.test/src/org/jboss/tools/cdk/ui/bot/test/server/editor/MinishiftServerEditorTest.java

```diff
@@ -0,0 +1,128 @@
+/******************************************************************************* 
+ * Copyright (c) 2018 Red Hat, Inc. 
+ * Distributed under license by Red Hat, Inc. All rights reserved. 
+ * This program is made available under the terms of the 
+ * Eclipse Public License v1.0 which accompanies this distribution, 
+ * and is available at http://www.eclipse.org/legal/epl-v10.html 
+ * 
+ * Contributors: 
+ * Red Hat, Inc. - initial API and implementation 
+ ******************************************************************************/
+package org.jboss.tools.cdk.ui.bot.test.server.editor;
+
+import static org.junit.Assert.assertTrue;
+
+import org.eclipse.reddeer.common.logging.Logger;
+import org.eclipse.reddeer.common.wait.TimePeriod;
+import org.eclipse.reddeer.common.wait.WaitUntil;
+import org.eclipse.reddeer.common.wait.WaitWhile;
+import org.eclipse.reddeer.eclipse.selectionwizard.NewMenuWizard;
+import org.eclipse.reddeer.eclipse.wst.server.ui.wizard.NewServerWizardPage;
+import org.eclipse.reddeer.workbench.core.condition.JobIsRunning;
+import org.jboss.tools.cdk.reddeer.core.condition.SystemJobIsRunning;
+import org.jboss.tools.cdk.reddeer.server.ui.CDKServersView;
+import org.jboss.tools.cdk.reddeer.server.ui.editor.CDKPart;
+import org.jboss.tools.cdk.reddeer.server.ui.editor.Minishift17ServerEditor;
+import org.jboss.tools.cdk.reddeer.server.ui.editor.MinishiftServerEditor;
+import org.jboss.tools.cdk.reddeer.server.ui.wizard.NewMinishiftServerWizardPage;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+/**
+ * Minishift's server editor tests
+ * @author odockal
+ *
+ */
+public class MinishiftServerEditorTest extends CDKServerEditorAbstractTest {
```

Comment:
Please add @RunWith(RedDeer.class) and other requirements if needed to run this class individually